vcxagency-node/src/service/entities/agent-connection/agent-connection.js

```diff
@@ -136,11 +137,22 @@ async function buildAgentConnectionAO (entityRecord, serviceWallets, serviceStor
       }
       logger.info(`${whoami} Handling message ${JSON.stringify(msgObject)}`)
       const resObject = await _handleAuthorizedAgentConnectionMsg(msgObject)
-      logger.debug(`${whoami} Sending response: ${JSON.stringify(resObject)}`)
+      _logResponseObject(resObject)
       return { response: resObject, shouldEncrypt: true }
     }
   }
 
+  function _logResponseObject(responseObject) {
```

Comment:
Most of the response messages are brief, so it's okay to print them, but for `MSGS` we would log all retrieved encrypted messages in debug - not very useful, better to log retrieved messages count in info.

Reply:
OK